config/pipelines/multiplexing_dil.wip.yml

```diff
@@ -0,0 +1,9 @@
+Multiplexing_Dil:
+  pipeline_group: Multiplexing
```

Comment:
Note: There is an existing pipeline_group with the same name. 
- This will add the relationships here to the same flow in the pipelines view. 
- In the other, Library Stock Plate goes to LB Lib Pool. On the sidebar, both suggested actions will be shown: Library Stock Plate -> Library Stock Dil Plate and Library Stock Plate -> LB Lib Pool